### DISCUSSION TITLE
Any e-mail through this solution

### DISCUSSION THREAD

**Discussion by bolshukhin** [KostyaEsmukov/smtp_to_telegram#19] [Ideas]
Hello. I think it would be interesting to finish the solution in such a way that other devices in the home network could send notifications to the telegram via this gateway

**Comment by KostyaEsmukov** [comment]
Could you elaborate?

I believe there's nothing preventing you from doing that as it is. If you need a queue or authentication then you can just place an exim4 relay in front of smtp_to_telegram.

**Reply by bolshukhin** [reply]
Hi. 

The idea is simple: I have a number of devices that send messages to e-mail. So I ask, how much is it possible to connect to the nas, so that using your solution for sending messages in telegram, you can send them from other devices inside the network?

Thanks

**Reply by KostyaEsmukov** [reply]
What have you tried so far to achieve this goal?
